Features.md

```diff
@@ -14,27 +14,308 @@
-Doc has creation timestamp | Checks if the sbom includes a creation timestamp | 
-Doc has require fields | Check if the sbom includes all of the required fields of the detected specification : <br>- **CycloneDX:** [bomFormat](https://cyclonedx.org/docs/1.4/json/#bomFormat), [SpecVersion](https://cyclonedx.org/docs/1.4/json/#specVersion), [Version](https://cyclonedx.org/docs/1.4/json/#version), [component:type](https://cyclonedx.org/docs/1.4/json/#components_items_type),[component:name](https://cyclonedx.org/docs/1.4/json/#components_items_name)<br>- **SPDX:** [CreationInfo](https://spdx.github.io/spdx-spec/v2.3/document-creation-information/), [Creator](https://spdx.github.io/spdx-spec/v2.3/document-creation-information/#68-creator-field), [Created](https://spdx.github.io/spdx-spec/v2.3/document-creation-information/#69-created-field), [SPDXVersion](https://spdx.github.io/spdx-spec/v2.3/document-creation-information/#61-spdx-version-field), [DataLicense](https://spdx.github.io/spdx-spec/v2.3/document-creation-information/#62-data-license-field), [SPDXIdentifier](https://spdx.github.io/spdx-spec/v2.3/document-creation-information/#63-spdx-identifier-field), [DocumentName](https://spdx.github.io/spdx-spec/v2.3/document-creation-information/#64-document-name-field), [DocumentNamespace](https://spdx.github.io/spdx-spec/v2.3/document-creation-information/#65-spdx-document-namespace-field), [PackageName](https://spdx.github.io/spdx-spec/v2.3/package-information/#71-package-name-field), [PackageSPDXIdentifier](https://spdx.github.io/spdx-spec/v2.3/package-information/#72-package-spdx-identifier-field), [PackageDowloadLocation](https://spdx.github.io/spdx-spec/v2.3/package-information/#77-package-download-location-field), [PackageVerificationCode](https://spdx.github.io/spdx-spec/v2.3/package-information/#79-package-verification-code-field) (if applicable) | 
-Doc has sharable license | Checks if the sbom includes an unemcumbered license that can aid in sharing | 
-Doc has creator tool name and version | If the sbom was created with a tool, it has the tool name and version | 
+
+# SBOM Quality Checks
+
+This page describes each SBOM Quality check in detail, including scoring criteria,
+remediation steps, and an explanation of the potential impact associated with a low score. 
+The checks are continually changing, and we welcome community feedback. 
+
+If you have ideas for additions or new detection techniques,
+please [contribute](https://github.com/interlynk-io/sbomqs#contributions)!
+
+## Taxonomy
+- A `Quality Check` is a test that can be performed on SBOM to return a binary result (e.g., A check for specification)
+- A `Quality Check Category` is a logical grouping of Quality Checks (e.g., "NTIA-Minimum-Elements" Checks)
+- A `Quality Check Set` is a collection of Quality Checks (e.g., "Default Check Set", "IoT Quality Set")
+
+## Scoring Methodology
+- Each Quality Check has an equal weight and a score range of 0.0 - 10.0.
```

Comment:
@surendrapathak eventually weights are going to be added which are user customizable. Should we indicate this.. or when its ready

Reply:
Added the Coming Soon.